nifi-bootstrap/src/main/java/org/apache/nifi/bootstrap/RunNiFi.java

```diff
@@ -993,6 +994,23 @@ public boolean accept(final File dir, final String filename) {
             cpFiles.add(file.getAbsolutePath());
         }
 
+        String runtimeJavaVersion = System.getProperty("java.version");
```

Comment:
On my phone but since commons lang3 is in the CP, SystemUtils can help determine if a version is at least IS_JAVA_11 or IS_JAVA_10. It does the parsing for you...may not help as I can't really review super well on the phone but it's a cool utility if it applies.

Reply:
In reviewing this, I looked at the SystemUtils that you mention. I kinda feel that it's "6 of one, half a dozen of the other." Either way, I think, is reasonable.

Reply:
If I recall correctly, I think the only reason I mentioned that was to avoid issues with specification vs implementation version that are taken into account by commons lang. java.specification.version can be different from java.version in cases where bug fixes are applied to a different JRE or usage of a newer JRE are downgraded by force.

Reply:
I think we're okay with the code as is.  This is just detecting if the JRE is 11 or newer, and adds the libs in the lib/java11 dir so that code has access to jaxb/activation/annotation libs; those libs were removed from the JDK/JRE as of Java 11.

Reply:
oh yeah I wasn't suggesting a follow on change. that bug will only arise in specific situations.